src/test/java/automationpractice/com/features/AddressFeatures.java

```diff
@@ -40,32 +38,32 @@ public void loggedInCustomerIsOnAddressPage() {
 
     @When("^he opens new address page and adds new address with proper data$")
     public void heOpensNewAddressPageAndAddsNewAddressWithProperData() {
+        final NewAddressData newAddressData = NewAddressInformations.NEW_ADDRESS_INFORMATIONS;
         addressPage.clickAddNewAddressButton();
-        newAddressPage.typeAddressFirstName();
-        newAddressPage.typeAddressLastName();
-        newAddressPage.typeAddress();
-        newAddressPage.typeCity();
-        newAddressPage.clickStateList();
-        newAddressPage.clickStateSelect();
-        newAddressPage.typePostalCode();
+        newAddressPage.typeFirstNameAddress(newAddressData.getFirstNameAddress());
+        newAddressPage.typeLastNameAddress(newAddressData.getLastNameAddress());
+        newAddressPage.typeAddress(newAddressData.getAddress());
+        newAddressPage.typeCity(newAddressData.getCity());
+        newAddressPage.typePostCode(newAddressData.getZipCode());
         newAddressPage.clickCountryList();
         newAddressPage.clickCountrySelect();
-        newAddressPage.typeMobilePhone();
-        newAddressPage.typeAddressName("MyAddress");
+        newAddressPage.typeMobilePhone(newAddressData.getMobilePhone());
+        newAddressPage.typeTitle(newAddressData.getNewAddresName());
+        newAddressPage.clickStateList();
+        newAddressPage.clickStateSelect(newAddressData.getState());
         newAddressPage.clickSaveNewAddressButton();
     }
 
     @Then("^new address is created$")
     public void newAddressIsCreated() {
-        final boolean isNewAddressNameDisplayed = addressPage.isNewAddressNameVisible();
-        assertThat(isNewAddressNameDisplayed).isTrue();
+        assertThat(addressPage.isNewAddressNameVisible()).isTrue();
     }
 
     @And("^new created address is deleted$")
     public void newCreatedAddressIsDeleted() {
-    addressPage.clickDeleteButton();
-        final boolean isNewAddressNameDisplayed = addressPage.isNewAddressNameVisible();
-        assertThat(isNewAddressNameDisplayed).isFalse();
+        addressPage.clickDeleteButton();
+        addressPage.getDriver().switchTo().alert().accept();
+        assertThat(addressPage.isNewAddressNameVisible()).isFalse();
     }
 
```

Comment:
addressPage.getDriver().switchTo().alert().accept(); to nie powinno byc w tym miejscu, stwórz metode
addressPage.acceptAlert(); i w niej getDriver().switchTo().alert().accept();